tensornetwork/backends/jax/jax_backend_test.py

```diff
@@ -669,6 +675,26 @@ def mv(x, H):
   compare_eigvals_and_eigvecs(
       np.stack(U, axis=1), eta, U_exact, eta_exact, thresh=1E-8)
 
+@pytest.mark.parametrize("dtype", [np.float64, np.complex128])
+@pytest.mark.parametrize("which", ["LR", "LM"])
+def test_eigs_large_matrix_with_init(dtype, which):
+  backend = jax_backend.JaxBackend()
+  D = 1000
+  np.random.seed(10)
+  init = backend.randn((D,), dtype=dtype, seed=10)
+  H = backend.randn((D, D), dtype=dtype, seed=10)
+
+  def mv(x, H):
+    return jax.numpy.dot(H, x)
+
+  eta, U = backend.eigs(
+      mv, [H], init, numeig=4, num_krylov_vecs=40, maxiter=500, which=which,
+      tol=1E-5)
```

Comment:
Is it ok that the tolerance is fixed independently of the dtype?

Reply:
in this case they are both double precision, so its fine. For lower precision one needs to change it, but this is currently not tested.